### PR TITLE
Define unacceptable postmortems

### PR DESCRIPTION
**Pull request by Jacob-Allebach**
## Description

Defines unacceptable postmortems with three examples.

### Type of Change

- [ ] Feature
- [ ] Bug fix
- [x] Documentation

## Contributors

- @Jacob-Allebach 

## Reminder

All GitHub Actions checks must be in a passing state in this branch or fork before this pull request can be merged.

### CODE REVIEW

**Review by PaigeCD — DISMISSED** [review]
I think this looks fine format-wise. As long as the other reviewers agree with the content, this is good to go.

**Review by anthonygc — DISMISSED** [review]
Looks good! Very consistent and clear!

**Review by Ochirsaikhan — COMMENTED** [review]
I think you should elaborate on the first point by giving examples of omitting failures. Is it deleting our commits or is it something different? Other than that, it looks good!

File: conduct.md [hunk 1]
```diff
@@ -110,9 +110,9 @@ TODO: Write an introductory paragraph.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should avoid omitting failures.
```
Comment (mariakimheinert) [inline conduct.md]: I agree with @Ochirsaikhan that it would be good to elaborate on what exactly "omitting failures" looks like!

**Review by mariakimheinert — CHANGES_REQUESTED** [review]

File: conduct.md (hunk 1 above)
Comment (Jacob-Allebach) [reply conduct.md]: I just added a bit more detail to that point to see if that's more clear now.

**Review by mariakimheinert — APPROVED** [review]
Looks good, @Jacob-Allebach! :shipit:

**Review by PaigeCD — APPROVED** [review]
👍

**Pull request merged**